Documentation/driver-api/soundwire/bra.rst

```diff
@@ -333,4 +333,4 @@ FIFO sizes to avoid xruns.
 
 Alignment requirements are currently not enforced at the core level
 but at the platform-level, e.g. for Intel the data sizes must be
-multiples of 32 bytes.
+equal to or larger than 32 bytes.
```

Comment:
if INTEL_BPT_MSG_BYTE_MIN is 16, why do you say 32 bytes here?

Reply:
My bad, it should be 16 bytes. Fixed it now.